metrics.yaml

```diff
@@ -9,11 +9,11 @@ startup_event:
     description: |
       The date and time the extension was started up
     bugs:
-      - https://linktobugs.page
+      - https://github.com/mozilla-extensions/firefox-bridge/issues/
     data_reviews:
-      - https://linktodatareviews.page
+      - https://github.com/mozilla-extensions/firefox-bridge/issues/87
```

Comment:
The data request must be approved and public.

Reply:
After speaking with Amir, the [data request](https://docs.google.com/document/d/1XC5qQnDECjKRpAeEiChuq4OfbPrrSdjOjF2zj2xhvHo/edit) was resolved in the comments of that document. He's following up with the glean team about this. The request was an internal ticket though, is there supposed to be a public request and public response?

If so, can we assume that #87 will host these and thus be the correct link for the metrics?